react-components/src/components/CogniteCadModelContainer/CogniteCadModelContainer.tsx

```diff
@@ -1,39 +1,51 @@
 import { useEffect, useRef } from "react";
-import { AddModelOptions, CogniteCadModel } from '@cognite/reveal';
+import { AddModelOptions, CogniteCadModel } from "@cognite/reveal";
 import { useReveal } from "../RevealContainer/RevealContext";
 import { Matrix4 } from "three";
 
 type Cognite3dModelProps = {
   addModelOptions: AddModelOptions;
   transform?: THREE.Matrix4;
-}
+};
 
-export default function CogniteCadModelContainer({ addModelOptions, transform }: Cognite3dModelProps) {
+export default function CogniteCadModelContainer({
+  addModelOptions,
+  transform,
+}: Cognite3dModelProps) {
   const modelRef = useRef<CogniteCadModel>();
   const viewer = useReveal();
   const { modelId, revisionId } = addModelOptions;
 
   useEffect(() => {
-    addModel(modelId, revisionId, transform)
-      .catch(console.error);
+    addModel(modelId, revisionId, transform).catch(console.error);
     return () => {
-      if (modelRef.current === undefined || !viewer.models.includes(modelRef.current)) return;
+      if (
+        modelRef.current === undefined ||
+        !viewer.models.includes(modelRef.current)
+      ) {
+        return;
+      }
       viewer.removeModel(modelRef.current);
       modelRef.current = undefined;
-    }
+    };
   }, [addModelOptions]);
 
-  if (modelRef.current !== undefined && transform !== undefined) {
+  useEffect(() => {
+    if (modelRef.current === undefined || transform === undefined) return;
```

Comment:
A little tempted to say that `undefined` transform should imply `transform = [0, 0, 0]`, but doubt it's really gonna be an issue, so whatever